### ISSUE TITLE
Plugin does not create MyUI.groovy at startup

### ISSUE THREAD

**Issue by philippneugebauer** [ondrej-kvasnovsky/grails-vaadin-plugin#50]
I was able to fix my first issue with the description in #48.

Unfortunately, although I was following the description at https://vaadin.com/wiki/-/wiki/Main/Vaadin+on+Grails+-+Create+project+in+IntelliJ+IDEA, there wasn't any `MyUI.groovy` in my Grails project after startup, so that I copied a simple one from a vaadin project and adapted it. Now, it does not show the vaadin design and e.g. buttons do not behave like buttons and are simple labels.

**Comment by christoph-frick** [comment]
the example code is no longer generated automatically.  there is a grails task/script to generate them manually.  if there is no styling on your app but it feels like in general it is working, there is a problem with the configured theme.  check with a tool like firebug on your client, what is happening and also check the server side logs for errors.  if nothing works at all it could be a problem with the communication between client and server, the widgetset, ...

**Comment by philippneugebauer** [comment]
Is there any documentation about this grails task? Can you update your documentation according to the changes?

**Comment by christoph-frick** [comment]
`grails help vaadin-quickstart`
 
    grails vaadin-quickstart -- Generates MyUI class and removed UrlMapping to easier project startup.

**Comment by philippneugebauer** [comment]
``` 
| Error Error running script vaadin-quickstart --stacktrace: java.util.regex.Pat
ternSyntaxException: Unexpected internal error near index 1
\
 ^ (NOTE: Stack trace has been filtered. Use --verbose to see entire trace.)
java.util.regex.PatternSyntaxException: Unexpected internal error near index 1
\
 ^
        at java_lang_String$split$7.call(Unknown Source)
        at VaadinQuickstart$_run_closure1_closure2.doCall(VaadinQuickstart:11)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
        at VaadinQuickstart$_run_closure1_closure2.call(VaadinQuickstart)
        at VaadinQuickstart$_run_closure1_closure2.call(VaadinQuickstart)
        at VaadinQuickstart$_run_closure1.doCall(VaadinQuickstart:9)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
        at org.codehaus.gant.GantMetaClass.invokeMethod(GantMetaClass.java:133)
        at org.codehaus.gant.GantBinding$_initializeGantBinding_closure5_closure
16_closure18.doCall(GantBinding.groovy:185)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
        at org.codehaus.gant.GantBinding$_initializeGantBinding_closure5_closure
16_closure18.doCall(GantBinding.groovy)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
        at org.codehaus.gant.GantBinding.withTargetEvent(GantBinding.groovy:90)
        at org.codehaus.gant.GantBinding.this$4$withTargetEvent(GantBinding.groo
vy)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
        at org.codehaus.gant.GantBinding$_initializeGantBinding_closure5_closure
16.doCall(GantBinding.groovy:185)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
        at org.codehaus.gant.GantBinding$_initializeGantBinding_closure5_closure
16.doCall(GantBinding.groovy)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
        at gant.Gant$_dispatch_closure5.doCall(Gant.groovy:381)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
        at gant.Gant$_dispatch_closure7.doCall(Gant.groovy:415)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
        at gant.Gant$_dispatch_closure7.doCall(Gant.groovy)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
        at gant.Gant.withBuildListeners(Gant.groovy:427)
        at gant.Gant.this$2$withBuildListeners(Gant.groovy)
        at gant.Gant$this$2$withBuildListeners$0.callCurrent(Unknown Source)
        at gant.Gant.dispatch(Gant.groovy:415)
        at gant.Gant.this$2$dispatch(Gant.groovy)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
        at gant.Gant.invokeMethod(Gant.groovy)
        at gant.Gant.executeTargets(Gant.groovy:591)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
        at gant.Gant.executeTargets(Gant.groovy:590)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
        at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(Refl
ectiveInterceptor.java:1270)
| Error Error running script vaadin-quickstart --stacktrace: java.util.regex.Pat
ternSyntaxException: Unexpected internal error near index 1
\
 ^
```
That's just what I get when I run the command.

I'm using grails 2.4.4 and this plugin in version 7.3.9

**Comment by christoph-frick** [comment]
you are on windows?  you can do what the script does manually by creating a file `src/groovy/app/MyUI.groovy` with the content from the template https://github.com/ondrej-kvasnovsky/grails-vaadin-plugin/blob/master/grails-vaadin7-plugin/src/samples/_MyUI.template

**Comment by philippneugebauer** [comment]
Yes, I am using windows

**Comment by ondrej-kvasnovsky** [comment]
Can you verify the fix with version: compile ":vaadin:7.4.7" please?
